utilities/ovs-pki.in

```diff
@@ -219,6 +219,10 @@ if test "$command" = "init"; then
         exit 1
```

Comment:
Could you update message above? Something like:

"$0: $pkidir already exists and --force not specified. --force will cause $pkidir to be permanently deleted."